src/search.cpp

```diff
@@ -611,10 +611,7 @@ Value Search::Worker::search(
 
     // Dive into quiescence search when the depth reaches zero
     if (depth <= 0)
-    {
-        constexpr auto nt = PvNode ? PV : NonPV;
-        return qsearch<nt>(pos, ss, alpha, beta);
```

Comment:
please dont change this https://github.com/official-stockfish/Stockfish/commit/4bc2a24245faa8ea8b3b8a80d03bd7436ac154a2